### PR TITLE
Migration to ASP.NET Core 6.0

### LINKED ISSUE #178: Migrate from ASP.NET Core 5.0 to 6.0
#### ✳️ Update .NET SDK version

Migration Guide -> https://docs.microsoft.com/en-us/aspnet/core/migration/50-to-60?view=aspnetcore-5.0&tabs=visual-studio

### PR DESCRIPTION
**Pull request by Jadhielv**
Closes #178

### CODE REVIEW

**Comment by Jadhielv** [comment]
@nminaya As soon as you've time, please check this PR

File: DWC.Blazor/DWC.Blazor.csproj [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <OutputType>Exe</OutputType>
     <LangVersion>7.3</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="5.0.7" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="6.0.3" />
```
Comment (nminaya) [inline DWC.Blazor/DWC.Blazor.csproj:10]: Have in mind that the other PR (#177) is modifying this same line

**Review by nminaya — APPROVED** [review]
@Jadhielv LGTM

File: DWC.Blazor/DWC.Blazor.csproj (hunk 1 above)
Comment (Jadhielv) [reply DWC.Blazor/DWC.Blazor.csproj:10]: yep, I got it

**Pull request merged**